handlers/analytics/handler.go

```diff
@@ -17,11 +17,12 @@ type searchHandler struct {
 
 // NewSearchHandler creates a new search handler
 func NewSearchHandler() (http.Handler, error) {
+	cfg, err := config.Get()
```

Comment:
You need to handle the possible error here. As you are only using a single config value I'd personally pass it in to the function - avoiding having to handle the error at all.

Reply:
Removed this line; updated `NewSearchHandler` to take the cfg values required (`SQSanalyticsURL`, `RedirectSecret`)